src/smtp/SMTP.ts

```diff
@@ -22,10 +24,13 @@ export default class SMTP {
 
 			if (!info.to.every(email => email.endsWith(`@${serverName}`))) { // if not all recipients are on this server
 				logger.log("Not all recipients are on this server. Will forward mail to other servers.")
-				logger.error("Forwarding mails to other servers is not implemented yet.")
 
+				// logger.error("Forwarding mails to other servers is not implemented yet.")
+
+				info.to.forEach(async mail => {
+					await smtpupstream.sendMail(info.from, mail, info.content)
+				})
```

Comment:
1. two loops (.every(...) and .forEach(...) (just use 1 for each)
2. you're forwarding the mails to ALL recipients, not just those on other domains

Reply:
1. ?
2. good point. Hmm.